src/Tracy/BlueScreen.php

```diff
@@ -68,6 +68,24 @@ public function render($exception)
 
 
 	/**
+	 * @param \Exception|\Throwable
+	 * @param string file path
+	 * @return void
+	 */
+	public function renderToFile($exception, $file)
+	{
+		if ($handle = @fopen($file, 'x')) { // @ file may already exist
```

Comment:
Ok but if I move that to Logger I must pass resource to this method from Logger or we should have "copy&paste" code in Logger and in this method. But file as resource for this method doesn't looks good for me (it's just feeling).

Do you have better idea?